tree/tree/inc/TTree.h

```diff
@@ -394,6 +397,7 @@ class TTree : public TNamed, public TAttLine, public TAttFill, public TAttMarker
    virtual Bool_t          GetImplicitMT() { return fIMTEnabled; }
    virtual Int_t          *GetIndex() { return &fIndex.fArray[0]; }
    virtual Double_t       *GetIndexValues() { return &fIndexValues.fArray[0]; }
+           ROOT::TIOFeatures GetIOFeatures() const;
```

Comment:
May add a using TIOFeatures = ROOT::TIOFeatures; above to short and align this line.